src/app/ubs-admin/components/ubs-admin-employee/ubs-admin-employee-table-mobile/ubs-admin-employee-table-mobile.component.html

```diff
@@ -0,0 +1,139 @@
+<div *ngIf="!isLoading; else spinner">
+  <div *ngIf="screenWidth < 480">
+    <mat-expansion-panel
+      *ngFor="let data of dataSource.filteredData"
+      [expanded]="chooseItemId === data.id"
+      (opened)="setChooseItem(data.id)"
+      hideToggle
+      #accordionItem
+    >
+      <mat-expansion-panel-header>
+        <mat-panel-title [style.visibility]="!accordionItem.expanded ? '' : 'hidden'">
+          {{ data.firstName }} {{ data.lastName }}
+        </mat-panel-title>
+        <mat-icon *ngIf="!accordionItem.expanded">keyboard_arrow_down</mat-icon>
+        <mat-icon *ngIf="accordionItem.expanded">keyboard_arrow_up</mat-icon>
+      </mat-expansion-panel-header>
+      <div class="employee-table-mobile">
+        <div id="edit-delete-btn">
+          <mat-icon class="mat-icon" (click)="openModal(data)">edit</mat-icon>
+          <mat-icon class="mat-icon" (click)="deleteEmployee(data.id)">delete</mat-icon>
+        </div>
+        <div class="employee-cell">
+          <div class="title">
+            <p><small> Ім'я працівника </small></p>
+          </div>
+          <div class="employee-info">
+            <p>{{ data.firstName }} {{ data.lastName }}</p>
+          </div>
+        </div>
+        <div class="employee-cell">
+          <div class="title">
+            <p><small> Посада </small></p>
+          </div>
+          <div class="employee-info">
+            <p *ngFor="let position of data.employeePositions">
+              {{ position.name }}
+            </p>
+          </div>
+        </div>
+
+        <div class="employee-cell">
+          <div class="title">
+            <p><small> Станція прийому </small></p>
+          </div>
+          <div class="employee-info">
+            <p *ngFor="let station of data.receivingStations">
+              {{ station.name }}
+            </p>
+          </div>
+        </div>
+
+        <div class="employee-cell">
+          <div class="title">
+            <p><small> email </small></p>
+          </div>
+          <div class="employee-info">
+            <p>
+              <a href="mailto:'data.email'"> {{ data.email }} </a>
+            </p>
+          </div>
+        </div>
+
+        <div class="employee-cell">
+          <div class="title">
+            <p><small> телефон </small></p>
+          </div>
+          <div class="employee-info">
+            <p>
+              <a href="tel:'data.phoneNumber'"> {{ data.phoneNumber }} </a>
+            </p>
+          </div>
+        </div>
+      </div>
+    </mat-expansion-panel>
+  </div>
+
+  <div *ngIf="screenWidth > 480">
```

Comment:
- So at first put that whole block for a large screen at the top, before the block for mobile.
- Then add `else` statement to be like `<div *ngIf="screenWidth > 480; else mobileView">`
- And replace that `*ngIf` statement for the mobile view block with just a local reference to look like `<div *ngIf="screenWidth < 480">` -> `<div #mobileView>`